### ISSUE TITLE
Undefined local variable or method map

### ISSUE THREAD

**Issue by particlebanana** [deis/fluentd#56]
I'm a little out of my element here but I ran into an issue where I wasn't seeing important logs in `deis logs` and looked into papertrail to see the following log from fluentd:

``` json
{
  "error": "#<NameError: undefined local variable or method `map' for #<Fluent::DeisOutput:00000000b12830>\nDid you mean?  tap>",
  "tag":"xxxxx.log",
  "message":"emit transaction failed: error_class=NameError error=\"undefined local variable or method `map' for #<Fluent::DeisOutput:00000000b12830>\\nDid you mean?  tap\" tag=\"xxxxx.log\""
}
```

If this doesn't seem like the correct repo for this issue let me know. From what I can tell there was some combination of unicode characters in that particular log that just didn't play nicely.

**Comment by jchauncey** [comment]
interesting. I haven't seen that before. What version of workflow are you using?

**Comment by particlebanana** [comment]
I'm currently running `2.5.0`. I haven't had time to do a deep dive yet but from a quick glance it seems a Node app is logging the following and that error appears:

``` js
console.log('- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -');
```

Very strange.

**Comment by jchauncey** [comment]
im wondering if that does something to jack up how we expect log messages to be delivered. if you had a good sample app i could use to debug that would be awesome. If not I will see what I can figure out.

**Comment by particlebanana** [comment]
That string may be a red herring. I created a sample app and logged that fine. I'll look what else is around that part of the logs and try and re-create it in a sample.

**Comment by jchauncey** [comment]
thanks.

**Comment by mboersma** [comment]
I tested this with workflow v2.5.0 and v2.6.0 using our simple [example-nodejs-express](https://github.com/deis/example-nodejs-express) app but couldn't make anything odd show up in fluentd logs. If you can make a reproducible sample, we will pounce on it.

**Comment by mboersma** [comment]
@particlebanana any luck making a sample app to help us reproduce this?

**Comment by particlebanana** [comment]
@jchauncey @mboersma sorry about that got busy and this slipped my mind. This repo definitely triggers it.

https://github.com/particlebanana/fluentd-char-test

**Comment by jchauncey** [comment]
k thanks we'll take a look

**Comment by mikermcneil** [comment]
@jchauncey it seems to be one of these:

`·` - on mac, **alt**+**shift**+**9**
`–` - on mac, **alt**+**-**
`«` - on mac, **alt**+**\**
`»` - on mac, **alt**+**shift**+**\**

**Comment by bacongobbler** [comment]
looks like this should be fixed via #74. If not please let me know and we can re-open this.
